web/server/vue-cli/e2e/pages/product.js

```diff
@@ -210,7 +211,8 @@ module.exports = {
     searchInput: ".v-toolbar__content input[type='text']",
     progressBar: ".v-data-table__progress",
     editBtn: "tr .edit-btn",
-    removeBtn: "tr .remove-btn"
+    removeBtn: "tr .remove-btn",
+    overlayScrim: "div .v-overlay__scrim"
```

Comment:
When you open a vuetify dialog window it will create two separate elements in the application root element:
- `v-overlay`
- `v-dialog`

![image](https://user-images.githubusercontent.com/6695818/140027823-4cc446f9-0f55-44a3-9c61-44a99124507a.png)

When you close a dialog, both of them will be dissapeared.

You class `v-overlay__scrim` is the child of the `v-overlay` element, and the `v-overlay` is connected to a `v-dialog` element.

So instead of this we have to find out which dialog is still in the DOM and have to wait for it while it is dissapeared instead of relying on an inner representation of a component.